app/views/leaderboards/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<% content_for :page_title, "Leaderboard: #{@quiz.title}" %>
+<% content_for :back_link, quizzes_path %>
+
+
```

Comment:
you may want to consider adding erb lint to your ci to clean up extra lines etc. in your views. It can autofix for you too 👍